src/Nethermind/Nethermind.WriteTheDocs/CliDocsGenerator.cs

```diff
@@ -85,7 +85,7 @@ public void Generate()
 
             Console.WriteLine(result);
             File.WriteAllText("cli.rst", result);
-            File.WriteAllText("../../../../../../docs/source/cli.rst", result);
```

Comment:
why did you need to change the folder structure? will it still work when invoked manually?

Reply:
I was getting System.IO errors while trying to run `dotnet run Nethermind.WriteTheDocs` manually. This fix removed those errors.